snapcraft.yaml

```diff
@@ -369,7 +452,8 @@ parts:
       # easily rerun.
       QUILT_PATCHES=$CRAFT_PROJECT_DIR/patches quilt push -a
       BUILD_DBGSYMS=false
-      if [ $CRAFT_ARCH_BUILD_FOR = "amd64" ] || [ $CRAFT_ARCH_BUILD_FOR = "arm64" ]; then
+      if [ $CRAFT_ARCH_BUILD_ON = "amd64" ]; then
+        # TODO: Update above comment?
```

Comment:
Which?

Reply:
Who knows :p

Reply:
@nteodosio So we would need to check if the patch is still required to be able to do builds on armhf hardware on launchpad

Reply:
Armhf doesn't build on core24 yet because of https://github.com/ubuntu/gnome-sdk/issues/240.

I think for now we could leave everything that is not bound with cross-compilation as is (that is, Github clean-ups and debug symbols), and then if deemed necessary or useful we can fine tune in a separate merge request.